tests/acceptance/test_external_studies.py

```diff
@@ -53,6 +55,18 @@ def test_external_study(name):
     # think this is sufficient for these tests which are intended to ensure we don't
     # accidentally break the API. If we're unable to execute a valid query, that's a
     # separate class of problem for which we need separate tests.
-    dataset = load_dataset_definition(dataset_def_path, user_args=())
+    with chdir(study_path):
```

Comment:
Could we not replace `chdir` with [pytest's `monkeypatch` fixture][1]?

```suggestion
    with monkeypatch.context() as m:
        m.chdir(study_path)
```

[1]: https://docs.pytest.org/en/stable/reference/reference.html#std-fixture-monkeypatch

Reply:
Oh yes, that saves some code. Thanks!